README.rst

```diff
@@ -47,6 +47,7 @@ virtual environment::
     $ cd OpenSlides
     $ python3 -m venv .virtualenv
     $ source .virtualenv/bin/activate
+    $ pip install -U setuptools
```

Comment:
I am not sure. Maybe it was that I had a fresh Ubuntu 14.04. Then the default virtualenv does not contain the current version of setuptools so something later failed. That's why I want to recommend to update setuptools in the step by step instruction.